docs/docs/integrating/container-engines.md

```diff
@@ -69,7 +68,6 @@ As a user, when container enrichment is enabled the event output will include en
         --pid=host --cgroupns=host --privileged \
         -v /etc/os-release:/etc/os-release-host:ro \
         -v /var/run/docker.sock:/var/run/docker.sock \
-        -e LIBBPFGO_OSRELEASE_FILE=/etc/os-release-host \
         aquasec/tracee:{{ git.tag }} \
         trace --output json --containers
```

Comment:
Seems like we still have the trace subcommand in some places in the documentation, which don't support anymore

Reply:
I'll raise other chore PR removing it.

Reply:
https://github.com/aquasecurity/tracee/pull/3238